multicluster_setup.adoc

```diff
@@ -0,0 +1,139 @@
+
+The same logic as above is applied to deploy the Member operator on the Member cluster:
+
+[source,bash]
+----
+$ export MEMBER_NS=toolchain-member-operator
+
+# on the member cluster, in the `toolchain-member-operator` namespace
+$ oc config use-context member-admin && MEMBER_NS=${MEMBER_NS} make get-member-operator-repo deploy-member
+...
+
+$ oc get pods -n ${MEMBER_NS}
+NAME                               READY   STATUS    RESTARTS   AGE
+member-operator-...-...            1/1     Running   0          ...
+----
+
+== Configuring the Operators
+
+Once Host and Member operators have been deployed on their respective cluster, they need to be configured to communicate with each other.
+
+At this stage, each operator has its own status resource, which is in a `Ready=False` status, because the host-member connection is not set yet:
+
+[source,bash]
+----
+# on the host cluster
+$ oc config use-context host-admin && oc get toolchainstatus -n ${HOST_NS}
+NAME               READY   LAST UPDATED
+toolchain-status   False   ...
+
+# on the member cluster
+$ oc config use-context member-admin && oc get memberstatus -n ${MEMBER_NS}
+NAME                      READY   LAST UPDATED
+toolchain-member-status   False   ...
+----
+
+[source,bash]
+----
+## on the member cluster
+
+# retrieve the API endpoint of the member cluster
+$ oc config use-context member-admin && oc get infrastructure cluster -o jsonpath='{.status.apiServerURL}' 
+https://api.<member_cluster_name>.devcluster.openshift.com:6443  
+
+# create/configure the ToolchainCluster resources on host and member clusters
+$ curl -sSL https://raw.githubusercontent.com/codeready-toolchain/toolchain-common/master/scripts/add-cluster.sh | bash -s -- -t member -mn ${MEMBER_NS} -hn ${HOST_NS}
+$ curl -sSL https://raw.githubusercontent.com/codeready-toolchain/toolchain-common/master/scripts/add-cluster.sh | bash -s -- -t host -mn ${MEMBER_NS} -hn ${HOST_NS}
+	
+
+# verify
+$ oc config use-context host-admin && oc get toolchainclusters -n ${HOST_NS}
+Switched to context "host-admin".
+NAME                               AGE   READY
+member-...                         ...   True
+
+$ oc config use-context member-admin && oc get toolchainclusters -n ${MEMBER_NS}
+Switched to context "member-admin".
+NAME                               AGE   READY
+host-...                           ...   True
+
+$ oc config use-context host-admin && oc get toolchainstatus -n ${HOST_NS} 
+Switched to context "host-admin".
+NAME               READY   LAST UPDATED
+toolchain-status   True    ...
+----
+
+At this point, Host and Member clusters are ready to use.
+
+[source,bash]
+----
+$ oc config use-context host-admin && oc project ${HOST_NS}
+Switched to context "host-admin".
+Now using project "toolchain-host-operator" on server ...
+
+$ HOST_NS=${HOST_NS} MEMBER_NS=${MEMBER_NS} operator-sdk test local ./test/e2e --no-setup --operator-namespace toolchain-e2e --verbose --go-test-flags "-timeout=30m -failfast -run TestE2E -v"
```

Comment:
btw, go 1.14 introduces the new format of the tests parameters and 1.15 won't support this old format
```suggestion
$ HOST_NS=${HOST_NS} MEMBER_NS=${MEMBER_NS} operator-sdk test local ./test/e2e --no-setup --operator-namespace toolchain-e2e --verbose --go-test-flags "-test.timeout=30m -test.failfast -run TestE2E -v"
```
so I guess that it's good to have it ready for the next version

Reply:
thanks for this suggestion @MatousJobanek